portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1182,6 +1182,14 @@ definition {
 			locator1 = "FormFields#GRID_OPTIONS_OPTION_ROW_NAME");
 	}
 
+	macro notViewOptionRowName {
```

Comment:
@gleicelisbino, is there any other macro using `not...`?

I think that would be interesting we name this macro as `viewOptionRowNameNotPresent`

Reply:
@natocesarrego I didn't find. Because, need use the path of Row and only exist with assertElementPresent. I will change the name.